configuration/pih/liquibase/liquibase.xml

```diff
@@ -154,4 +154,37 @@
         </sql>
     </changeSet>
 
-</databaseChangeLog>
+    <changeSet id="20210602-delete-unused-ces-programs" author="jmbabazi">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+            select count(*) from patient_program where program_id in
+            (select program_id from program where uuid in ("9fe0c32c-981f-4283-b4f4-168285a52c7d",
+                "40657f91-5991-44fe-a16f-220dbeb9dcbd", "1eb13158-f39c-444a-b77c-78f1d78123be") and retired = 0);
+            </sqlCheck>
+        </preConditions>
+        <comment>If the count is 0, then remove these programs (Hypertension program, Diabetes program, Epilepsy program)</comment>
+        <sql>
+            -- void patient states related to these programs
```

Comment:
This does not make sense to me @jmbabazi .  We can't void states, and there won't be states anyway.  If the precondition passes (there are no patient programs), then by definition there can't be any patient states associated with these programs, right?  And if there were, voiding them and then deleting the states in the next statement doesn't work.